CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+# Change Log - @itwin/insights-client
+
+## 0.3.3
+Tue, 15 Nov 2022
+
+### Minor changes
```

Comment:
```suggestion
### Patches
```

This is only a patch version bump, so I think it wouldn't be correct to call this `Minor changes`.